server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -1943,6 +1959,10 @@ public boolean useTimeSeriesSyntheticId() {
         return useTimeSeriesSyntheticId;
     }
 
+    public boolean isUseTimeSeriesDocValuesCodec() {
```

Comment:
Let's just call it `useTimeSeriesDocValuesCodec`?  `isUse` reads a bit strangely to me.